packages/mechanic-engine-react/index.js

```diff
@@ -74,9 +70,8 @@ export default _default;
     return;
   }
 
-  reactHotLoader.register(root, "root", "/Users/fdoflorenzano/Projects/DSI/mechanic/packages/mechanic-react-engine/src/index.js");
-  reactHotLoader.register(run, "run", "/Users/fdoflorenzano/Projects/DSI/mechanic/packages/mechanic-react-engine/src/index.js");
-  reactHotLoader.register(_default, "default", "/Users/fdoflorenzano/Projects/DSI/mechanic/packages/mechanic-react-engine/src/index.js");
+  reactHotLoader.register(root, "root", "/Users/fdoflorenzano/Projects/DSI/mechanic/packages/mechanic-engine-react/src/index.js");
```

Comment:
shouldn't this url be relative to the project root?

Reply:
Commented on this issue on #20 😄